### ISSUE TITLE
don't force winetricks silent mode

### ISSUE THREAD

**Issue by sixsixfive** [lutris/lutris#317]
runninq winetricks in silent mode breaks games.

Silent mode might work for very simple dll overrides but it won't work for complex apps like xna or dot net

examples: 

* https://lutris.net/games/install/2279/view
* https://lutris.net/games/revolver360-reactor/

both games use xna, however since lutris calls winetricks in silent mode which makes it unable to accept the Xna and dotnet EULAs winetricks won't create the needed dll overrides so the games fail to launch.

**Comment by Xodetaetl** [comment]
There's an undocumented option to turn it off. No time to look it up or document it right now but it should simply be `silent: false`

**Comment by strycore** [comment]
That should do it :D

**Comment by sixsixfive** [comment]
also non-quiet mode should be the default behavior and silent should be an option

**Comment by strycore** [comment]
winetricks apps that fail with silent mode are an exception, I've personally never encountered such a case, and this includes numerous install of .NET through Lutris installers.

**Comment by sixsixfive** [comment]
yes, but you never know. 

winetricks changes almost any day and the files are not hosted by them so they might change in the future. Also making silent the default behavior is bad since the most people will never realize why the game won't work(remember there is no error message so the user thinks everything was successfully installed and will blame lutris for it...)

**Comment by strycore** [comment]
I agree that stuff may break in the future but we'll act accordingly when that happens.
What I don't like doing is introducing code "just in case", I trust things that are proven to work and fix things that are proven to be broken. No one knows what the future will be made of. 

Speaking of stuff that is proven to be working, I've made an installer for the Humble Bundle version of Aqua Kitty, both .NET and XNA are installed silently and the game works great! (minus fullscreen issues in Unity, but it works fine in Gnome)

Anyway, I know that at some point we had a problem with Winetricks not working properly and I don't remember when we fixed it. The current master branch should be considered to be a Release Candidate for 0.3.8 so maybe try with that and see if the silent issues were specific to our previous version.
